MANIFEST.in

```diff
@@ -1 +1,11 @@
-recursive-include docker-intermine-gradle *
+include docker-intermine-gradle/ *.yml
+
+include docker-intermine-gradle/intermine-builder/ *.sh *.Dockerfile
+
+include docker-intermine-gradle/postgres/ *.conf *.sh *.Dockerfile docker-healthcheck
+
+include docker-intermine-gradle/solr/ *.Dockerfile
+include docker-intermine-gradle/solr/scripts/ *.sh
+
+include docker-intermine-gradle/tomcat/ *.Dockerfile
+include docker-intermine-gradle/configs/ *.xml
```

Comment:
Looks like `configs` should be inside the `tomcat` folder.

Also could you do a recursive-include of each dockerfile folder? Meaning `recursive-include docker-intermine-gradle/intermine-builder *` and so on if that works. It would be useful in case the contents of the dockerfiles change. Sorry for being unclear! :sweat_smile:

Reply:
ok, just to be clear.

recursive-include should be for docker-intermine-gradle/intermine-builder

but for others include is fine?

Reply:
Thanks for confirming. recursive-include should be for all of the docker containers: intermine_builder, postgres, solr and tomcat.

Reply:
done.